packages/search-spa/set-env.ts

```diff
@@ -7,7 +7,7 @@ const targetPath = './src/environments/environment.ts';
 require('dotenv').config();
 
 const envConfigFile = `export const environment = {
-   API_URL: process.env.API_URL,
+   API_URL: '${process.env.API_URL}',
```

Comment:
Use a fallback when using interpolation in javascript templates for potentially undefined values.
```suggestion
   API_URL: '${process.env.API_URL || ''}',
```

Reply:
The above line can cause problems down the line if API_URL is undefined.
```ts
console.log(typeof a)
// > undefined

console.log(typeof `${a}`)
// > string
```